requirements/pip.txt

```diff
@@ -78,12 +78,6 @@ django-crispy-forms==1.13.0
 
 docker==5.0.3
 
-django-textclassifier==1.0
-# django-textclassifier doesn't have pinned versions
-# if there is an update they could break our code
-nltk==3.6.6
-textblob==0.17.1
```

Comment:
These are big dependencies, so 👍 on getting rid of them if we aren't using them.